### ISSUE TITLE
Improve informations about the current topology when adding land informations

### ISSUE THREAD

**Issue by LePetitTim** [GeotrekCE/Geotrek-admin#3305]
- Add informations about the current topology (here 10439)

![2022-10-18_11-59](https://user-images.githubusercontent.com/26329336/196399766-bac07a80-3160-439e-beaa-be0c8d21e7c7.png)
![2022-10-18_11-59_1](https://user-images.githubusercontent.com/26329336/196399777-69c1cf9c-886d-4502-a406-976c0a0cf49e.png)

- Remove possibility to move start and end points + remove cursor ![2022-10-18_12-00](https://user-images.githubusercontent.com/26329336/196400113-ad974c98-9bf5-4af7-a8ba-cca6b0a536c6.png)

- Show the topology

**Comment by camillemonchicourt** [comment]
I dont see in which case we can add a land information only on one path ? Only on one topology ?

**Comment by LePetitTim** [comment]
We can add land information on one topology

**Comment by marcantoinedupre** [comment]
The feature is available on the details page of every topological object.

![image](https://user-images.githubusercontent.com/15977920/196417993-83409afc-c1d4-49d9-b874-272850a9ab5d.png)

**Comment by LePetitTim** [comment]
Exactly, but we do not have information about which topology we are on.
It would be better to add some information about the topology somewhere

**Comment by camillemonchicourt** [comment]
Yes, initially when we added an intervention on an existing topology the map was blocked and a text was displayed on top left of the map saying "on signage 14" or something like this but it seems to dont work anymore
